From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Sat, 4 Oct 2014 12:17:30 -0400
Subject: adjust setup.py

hardcode version
remove -Werror
don't install pvtree to /usr/bin
---
 setup.py | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/setup.py b/setup.py
index ec13e77..f2781df 100755
--- a/setup.py
+++ b/setup.py
@@ -9,9 +9,6 @@ try:
     from setuptools import setup, Extension
 
     setup_args = dict(
-        entry_points = {
-            'console_scripts': [
-                'pvtree.py = cothread.tools.pvtree:main' ] },
         install_requires = ['numpy'],
         zip_safe = False)
 
@@ -20,13 +17,9 @@ except ImportError:
     setup_args = {}
 
 
-# these lines allow the version to be specified in Makefile.RELEASE
-version = os.environ.get('MODULEVER', 'unknown')
-
 # Extension module providing core coroutine functionality.  Very similar in
 # spirit to greenlet.
 extra_compile_args = [
-    '-Werror',
     '-Wall',
     '-Wextra',
     '-Wno-unused-parameter',
@@ -53,7 +46,7 @@ if platform.system() == 'Windows':
 
 setup(
     name = 'cothread',
-    version = version,
+    version = '2.12',
     description = 'Cooperative threading based utilities',
     author = 'Michael Abbott',
     author_email = 'Michael.Abbott@diamond.ac.uk',
